tests/org.eclipse.passage.loc.features.core.tests/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.eclipse.passage.loc.features.core.tests
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: org.eclipse.passage.loc.features.core.tests
+Bundle-Version: 0.1.0.qualifier
+Bundle-Name: %Bundle-Name
+Bundle-Vendor: %Bundle-Vendor
+Bundle-Copyright: %Bundle-Copyright
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Require-Bundle: org.junit;bundle-version="4.12.0",
+ org.eclipse.passage.loc.features.core;bundle-version="0.5.0",
+ org.mockito
```

Comment:
Can we [avoid mocks](https://www.yegor256.com/2014/09/23/built-in-fake-objects.html)?